backend/sumo-championship/src/main/java/com/sumoc/sumochampionship/api/dto/websiteuser/WebsiteUserRequest.java

```diff
@@ -16,4 +16,6 @@ public class WebsiteUserRequest {
     String lastname;
     String password;
     UserRole userRole;
+    String club; // Optional field for club
+    String country; // Optional field for country
```

Comment:
I have created Country enum. You should use it here. Additionally club should be stored as Long (id)